src/components/chart/component.jsx

```diff
@@ -182,15 +182,16 @@ function Chart({ className, data, config, metadata }) {
               />
             ))}
           {lines &&
-            lines.map(line => (
-              <Line
-                type="monotone"
-                key={line.key}
-                dataKey={line.key}
-                stroke={line.color}
-                strokeWidth={3}
-                dot={CustomDot}
-              />
+            lines.map((line, n) => (
+              <Fragment key={n}>
```

Comment:
nice one 👍

Reply:
@blayhem yes it looks strange ))) I have returned line.key instead of n. 
The Line component returns warning - (non unique key prop). This warning - inside component and i don't have access to it. So I save a Fragment https://github.com/Vizzuality/copernicus-forest/commit/744c73e518dc0345d6fe70dde283f41061fdbb09